File: src/targets/javascript/fetch/fixtures/jsonObj-multiline.js
const url = 'https://httpbin.org/anything';
const options = {
  method: 'POST',
  headers: {'content-type': 'application/json'},
  body: JSON.stringify({foo: 'bar'})
};

try {
  const response = await fetch(url, options);
  const data = await response.json();
  console.log(data);
} catch (error) {
  console.error(error);
}

File: src/targets/ruby/faraday/fixtures/jsonObj-multiline.rb
require 'faraday'

conn = Faraday.new(
  url: 'https://httpbin.org',
  headers: {'Content-Type' => 'application/json'}
)

response = conn.post('/anything') do |req|
  req.body = "{\n  \"foo\": \"bar\"\n}"
end

puts response.status
puts response.body